src/dashboard/messages.ts

```diff
@@ -0,0 +1,16 @@
+export default {
+    REGISTRATION_CANCELLED:                   'The setup process has been aborted.',
+    REGISTRATION_ENTER_EMAIL_INVITATION:      'Log into dashboard.testcafe.io to start the setup process.\nEnter your email to receive a login link.',
```

Comment:
```suggestion
    REGISTRATION_ENTER_EMAIL_INVITATION:      'Welcome to the TestCafe Dashboard setup wizard.\n This wizard requires an Internet connection.\n Enter your email address to receive a TestCafe Dashboard authentication link.\nIf you don't have a TestCafe Dashboard account, the link will prompt you to sign up.',
```

the logic was off: we invite users to log into the website, but then request their email (what for? I have already opened the browser after reading the first line..) this needs to be the other way around